libs/partners/google-genai/pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[tool.poetry]
+name = "langchain-google-genai"
+version = "0.0.1"
+description = "An integration package connecting Google's genai package and LangChain"
+authors = []
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4.0"
```

Comment:
Had to increase the min version since google-generativeai doesn't permit python 3.8 fyi